server.py

```diff
@@ -1,103 +1,150 @@
 
 logging.basicConfig(level=settings.LOGGING_LEVEL, format=settings.LOGGING_FORMAT)
 logger = wrap_logger(logging.getLogger(__name__))
+
 app = Flask(__name__)
-app.config['MONGODB_URL'] = settings.MONGODB_URL
-publisher = Publisher(logger)
+app.config['SQLALCHEMY_DATABASE_URI'] = settings.DB_URI
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = settings.SQLALCHEMY_TRACK_MODIFICATIONS
+
+db = SQLAlchemy(app=app)
 
 schema = Schema({
-    'survey_id': str,
+    'added_ms': Coerce(int),
     'form': str,
-    'ru_ref': str,
+    'page': All(Coerce(int), Range(min=1)),
     'period': str,
-    'added_ms': Coerce(int),
     'per_page': All(Coerce(int), Range(min=1, max=100)),
-    'page': All(Coerce(int), Range(min=1))
+    'ru_ref': str,
+    'survey_id': str,
 })
 
 
+class InvalidUsageError(Exception):
+    status_code = 400
+
+    def __init__(self, message, status_code=None, payload=None):
+        Exception.__init__(self)
+        self.message = message
+        if status_code is not None:
+            self.status_code = status_code
+        self.payload = payload
+
+    def to_dict(self):
+        rv = dict(self.payload or ())
+        rv['message'] = self.message
+        return rv
+
+
+class SurveyResponse(db.Model):
+    __tablename__ = 'responses'
+    tx_id = db.Column("tx_id",
+                      UUID,
+                      primary_key=True)
+
+    ts = db.Column("ts",
+                   db.TIMESTAMP(timezone=True),
+                   server_default=db.func.now(),
+                   onupdate=db.func.now())
+
+    invalid = db.Column("invalid",
+                        db.Boolean,
+                        default=False)
+
+    data = db.Column("data", JSONB)
+
+    def __init__(self, tx_id, invalid, data):
+        self.tx_id = tx_id
+        self.invalid = invalid
+        self.data = data
+
+    def __repr__(self):
+        return '<SurveyResponse {}>'.format(self.tx_id)
+
+    def to_dict(self):
+        return {c.name: getattr(self, c.name) for c in self.__table__.columns}
+
+
 def _get_value(key):
     value = os.getenv(key)
     if not value:
         raise ValueError("No value set for " + key)
 
 
 def check_default_env_vars():
+    env_vars = ["RABBIT_CS_QUEUE", "RABBIT_CTP_QUEUE", "RABBIT_CORA_QUEUE",
+                "RABBITMQ_HOST", "RABBITMQ_PORT", "RABBITMQ_DEFAULT_USER",
+                "RABBITMQ_DEFAULT_PASS", "RABBITMQ_DEFAULT_VHOST",
+                "RABBITMQ_HOST2", "RABBITMQ_PORT2",
+                ]
 
-    env_vars = ["MONGODB_URL", "RABBIT_CS_QUEUE", "RABBIT_CTP_QUEUE", "RABBIT_CORA_QUEUE",
-                "RABBITMQ_HOST", "RABBITMQ_PORT", "RABBITMQ_DEFAULT_USER", "RABBITMQ_DEFAULT_PASS",
-                "RABBITMQ_DEFAULT_VHOST", "RABBITMQ_HOST2", "RABBITMQ_PORT2"]
-
+    missing = False
     for i in env_vars:
         try:
             _get_value(i)
         except ValueError as e:
             logger.error("Unable to start service", error=e)
-            missing_env_var = True
+            missing = True
```

Comment:
I raised the same issue with @gavinleeedwards - it makes sense from a startup point of view, as you only need to fail once to understand exactly what envvars are missing.